docs/ui/insight/get-started.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Get started with RedisInsight
+linkTitle: Get started
+description: Get started with RedisInsight
+weight: 40
+
+---
+
+RedisInsight is available as a [standalone download](https://redis.com/redis-enterprise/redis-insight/) or [bundled with Redis Stack](/docs/stack/get-started/install). Install it and try the in-app examples available in Workbench.
+
+
+## Install RedisInsight 
+
+You can use RedisInsight on MacOS, Linux, or Windows.
+
+**Install on MacOS**
+
+Redis offers RedisInsight on Intel and M1. 
+
+1. Download [RedisInsinght](https://redis.com/redis-enterprise/redis-insight) or [Redis Stack](/docs/stack/get-started/install/mac-os). 
```

Comment:
Linux users also can download it from Snapcraft (https://snapcraft.io/redisinsight) or FlatHub (https://flathub.org/apps/details/com.redis.RedisInsight)